### PR TITLE
IguanaDEX & IguanaDEX v2

### PR DESCRIPTION
**Pull request by iguanadex**
##### Name (to be shown on DefiLlama):

IguanaDEX

##### Twitter Link:

@IguanaDEX

##### List of audit links if any:

https://github.com/peckshield/publications/blob/master/audit_reports/PeckShield-Audit-Report-PancakeSwapV3-v1.0.pdf
https://github.com/slowmist/Knowledge-Base/blob/master/open-report-V2/smart-contract/SlowMist%20Audit%20Report%20-%20PancakeSwap_v3_en-us.pdf
https://dapp.org.uk/reports/uniswapv2.html
https://github.com/slowmist/Knowledge-Base/blob/master/open-report/Smart%20Contract%20Security%20Audit%20Report%20%20-%20PancakeSwap.pdf

##### Website Link:

https://iguanadex.com

##### Logo (High resolution, will be shown with rounded borders):

https://iguanadex.com/images/token_logo.png

##### Current TVL:

USD 210,000

##### Chain:

Etherlink

##### Short Description (to be shown on DefiLlama):

One-stop decentralized trading on Etherlink

##### Category (full list at https://defillama.com/categories) *Please choose only one:

Dexes

##### forkedFrom (Does your project originate from another project):

PancakeSwap/Uniswap v2 & v3

##### methodology (what is being counted as tvl, how is tvl being calculated):

USD value currently deposited in (v3) pools.

##### Github org/user (Optional, if your code is open source, we can track activity):

iguanadex (Github user)

### CODE REVIEW

**Comment by llamatester** [comment]
The adapter at projects/iguanadex-v2 exports TVL: 
        
 
 
    etlk                      0
    
    total                    0

**Comment by llamatester** [comment]
Error while running adapter at projects/iguanadex-v3: 
        
 
  
    
    
    [object Object]

File: projects/iguanadex-v3/index.js
Comment (g1nt0ki) [inline projects/iguanadex-v3/index.js:1]: I am having trouble pulling pair creation events from etherlink RPC, do you have graph api that we can query for the logs?

Comment (iguanadex) [reply projects/iguanadex-v3/index.js:1]: v3: https://api.studio.thegraph.com/query/86688/exchange-v3-etherlink/version/latest
v2: https://api.studio.thegraph.com/query/86688/exchange-v2-etherlink/version/latest

Comment (g1nt0ki) [reply projects/iguanadex-v3/index.js:1]: thanks @iguanadex

**Comment by llamatester** [comment]
The adapter at projects/wagmi-com exports TVL: 
        
 
 
    staking                   2.19 M
    kava                      1.84 M
    kava-staking              1.83 M
    metis                     1.34 M
    iotaevm                   654.88 k
    metis-staking             361.50 k
    ethereum                  193.12 k
    era                       41.30 k
    zklink                    14.04 k
    fantom                    664
    base                      580
    
    total                    4.09 M

**Comment by llamatester** [comment]
The adapter at projects/iguana-v2 exports TVL: 
        
 
 
    etlk                      0
    
    total                    0

**Comment by llamatester** [comment]
The adapter at projects/iguana-v3 exports TVL: 
        
 
 
    etlk                      140.95 k
    
    total                    140.95 k

**Pull request merged**